src/clients/api/queries/getPendingRewards/formatOutput/index.ts

```diff
@@ -14,14 +19,17 @@ import formatToMainPoolPendingRewardGroup from './formatToMainPoolPendingRewardG
 
 const formatOutput = ({
   contractCallResults,
+  rewardTokenPrices,
 }: {
```

Comment:
Nit: I'd use an object to define these params, so that it can be easily expanded later on if needed.